oak-jcr/src/test/resources/logback-test.xml

```diff
@@ -15,6 +15,8 @@
    limitations under the License.
   -->
 <configuration>
+    <!-- disable logback status output filling up stdout before tests run -->
+    <statusListener class="ch.qos.logback.core.status.NopStatusListener" />
```

Comment:
These log config changes were added for debugging and working on the `HttpBinaryIT` (formerly named `URLBinaryIT`. This change here removed a lot of annoying warnings of logback at the start, which messed up the log view in IntelliJ's test runner.

It's not important and can be removed if desired.